packages/web-app-admin-settings/src/components/Spaces/SpacesList.vue

```diff
@@ -475,9 +475,21 @@ export default defineComponent({
   .oc-table-data-cell-usedQuota {
     display: none;
 
-    @media only screen and (min-width: 1700px) {
+    @media only screen and (min-width: 1600px) {
       display: table-cell;
     }
   }
+  &-squashed {
+    .oc-table-header-cell-totalQuota,
+    .oc-table-data-cell-totalQuota,
+    .oc-table-header-cell-availableQuota,
+    .oc-table-data-cell-availableQuota,
```

Comment:
I believe `availableQuota` does not exist anymore, it was renamed. In line 472/473 as well.

```suggestion
```